README.md

```diff
@@ -140,4 +142,4 @@ snakemake \
 
 *   `RNAseq_demultiplex.sh`
 
-    Demultiplexes samples based on i7 barcodes. If several plates (different i5) were sequenced together, an initial demultiplex step has to be done using Lance's i5 code (i5_parse_gencomp1_template.sbatch)
+    It demultiplexes samples based on i7 barcodes. If several plates (different i5) were sequenced together, an initial demultiplexing step has to be done using i5 barcodes (custome code: i5_parse_gencomp1_template.sbatch)
```

Comment:
I'm not sure it's true that we need a separate demultiplexing step if an i5 was included. It would be a standard dual-barcode situation there, and can be demultiplexed by most sequencing centers.

Reply:
I don't think we should have the demultiplexing part here anyways. Either we add proper code for it, or we just let people take care of that on their own. As you said, the demultiplexing will most probably be done by the sequencing centers. I've removed the "additional scripts" part for now. We  might decide to add something later, but maybe not necessary right now.